### ISSUE TITLE
Missing dvb-usb-af9035-02.fw firmware

### ISSUE THREAD

**Issue by WallyCZ** [osmc/osmc#193]
To get my USB DVB-T stick (avertv volar hd) to work, I had to download it manually. It would be nice to have it included in distro (in openelec it works without this step).

cd /lib/firmware
sudo wget http://xgazza.altervista.org/Linux/DVB/dvb-usb-af9035-02.fw

**Comment by dam09fr** [comment]
+1

**Comment by mcurly** [comment]
hello @WallyCZ and @dam09fr and sorry if I'm digging up an ancient thread.
I've recently tried to use this device of ours as an infrared receiver in osmc in pi 1. I was able to do it on Arch, both pi 2 and x86_64. But not quite here in osmc. running inside pi 2.

Has anyone of you managed to use your device as an infrared receiver, inside osnc, and are willing to hand over some insights into the subject?

thanks in advance.

**Comment by samnazarko** [comment]
How does it work as an infrared receiver on Arch? Does it expose a /dev/lirc node?

**Comment by mcurly** [comment]
hi @samnazarko  no it exposes itself as a /dev/input/eventX device

**Comment by mcurly** [comment]
the same as in osmc actually

**Comment by samnazarko** [comment]
You can likely set it up with eventlircd then.

You didn't really explain how you set it up in Arch, so can't comment there, but eventlircd will likely let you achieve this.

**Comment by mcurly** [comment]
@samnazarko Hello, Sam!
First, I am sorry for not replying earlier.
From late March I went on to arch-linux-arm up until (I think) it was September or so, when kodi stopped working after some upgrade and the issue lingered on for several days, and I don't know yet if they solved it already, only then I came back to osmc. And didn't have the sollution for using the usb dvb-t dongle as a ir receiver.
But I finally have managed to do it.
I followed this wiki [https://wiki.libreelec.tv/infrared_remotes?s[]=eventlircd](url) almost to the letter.
The sole thing that missed (in osmc) was the default installation of ir-keytable.
When that was sort out the goal was met. Finally it worked.
Presently, it's in use in both osmc pi2 and osmc pi3.
I tried to contact you in the IRC, but up until now I couldn't.
I am available to make a tut on it and hand it over to you for reviewing or publication (if you think it would be of help).

Thank you. Cheers!
